runtime/lua/vim/uri.lua

```diff
@@ -76,6 +76,10 @@ local function uri_from_bufnr(bufnr)
 end
 
 local function uri_to_fname(uri)
+  local scheme = assert(uri:match("^([a-z]+)://.*"), 'Uri must contain a scheme: ' .. uri)
```

Comment:
nit
```suggestion
  local scheme = assert(uri:match("^([a-z]+)://.*"), 'URI must contain a scheme: ' .. uri)
```

Reply:
The scheme may have  also digit, "+", "-" and ".".
https://tools.ietf.org/html/rfc3986#page-17